src/lwc/geFormFieldLookup/geFormFieldLookup.js

```diff
@@ -1,16 +1,18 @@
 import { LightningElement, api, wire, track } from 'lwc';
 import { getObjectInfo } from 'lightning/uiObjectInfoApi';
-import { getRecord, getFieldValue } from 'lightning/uiRecordApi';
+import { getRecord } from 'lightning/uiRecordApi';
 import doSearch from '@salesforce/apex/GE_LookupController.doSearch';
 import { isNotEmpty } from 'c/utilCommon';
+import { handleError } from 'c/utilTemplateBuilder';
 
 const DELAY = 300;
```

Comment:
Not for this PR/story, but we should think long term how to avoid timing-based delays and setTimeout for a future improvement